src/mainwindow.cpp

```diff
@@ -8948,9 +8948,15 @@ void MainWindow::handleScriptingNotesTagUpdating() {
 
     // workaround when signal blocking doesn't work correctly
     directoryWatcherWorkaround(true, true);
-
-    const QVector<Note> &notes = Note::fetchAll();
+    const QString defExt = Note::defaultNoteFileExtension();
+    const QStringList exts = Note::customNoteFileExtensionList(QStringLiteral("."));
+    const QVector<Note> notes = Note::fetchAll();
     for (const Note &note : notes) {
+        // ignore non-note files
+        const auto suffix = note.fileNameSuffix();
+        if (defExt != suffix || !exts.contains(note.fileNameSuffix())) {
```

Comment:
Is there a reason you are using `note.fileNameSuffix();` twice?

Reply:
And doesn't `Note::customNoteFileExtensionList` already contain `Note::defaultNoteFileExtension()`?

Reply:
changed.

No idea about whether its included or not.

Reply:
> No idea about whether its included or not.

Don't you want to test that before we include broken code? 😆

Reply:
checked, both are different for me. `custom...()` is empty, the other one isn't.

Reply:
Now that's weird... It should be included (and was for me)...

Reply:
But still: https://github.com/pbek/QOwnNotes/pull/2310#discussion_r756094020

Reply:
putting both in a `qWarning()<<..` gives:

```
Warning: Extensions "md" () ((null):0, (null)
```
notice the empty ()

Reply:
How did you manage that? 😆 

https://github.com/pbek/QOwnNotes/blob/bacd3b8e80d608c9dfab0c8dfb7160343d196bef/src/services/databaseservice.cpp#L872-L883

...should add at least md and txt.

Reply:
And you shouldn't be able to remove all extensions in the settings.